app/src/main/java/org/kiwix/kiwixmobile/nav/destination/reader/KiwixReaderFragment.kt

```diff
@@ -263,6 +264,18 @@ class KiwixReaderFragment : CoreReaderFragment() {
     )
   }
 
+  override fun onToggledFullscreen(isFullScreen: Boolean) {
+    if (isFullScreen) {
+      requireActivity().bottom_nav_view.visibility = GONE
+      setParentFragmentsBottomMarginTo(0)
+      getCurrentWebView().translationY = 0f
+    } else {
+      requireActivity().bottom_nav_view.visibility = VISIBLE
+      val fullScreenFlag = WindowManager.LayoutParams.FLAG_FULLSCREEN
```

Comment:
I don't see why we need to clear this flag? The WebViewClient should and has been managing the flags.

I would expect you would want to mimic the functionality of `fun closeFullScreen() {` here, similar to comment above I would want aa method to reduce duplication